lib/l10n/app_en.arb

```diff
@@ -0,0 +1,60 @@
+{
+  "chooseAccountPageTitle": "Choose account",
+  "@chooseAccountPageTitle": {
+    "description": "Title for ChooseAccountPage"
```

Comment:
branch-structure nit: Let's add each of these translations to this .arb file in the same commit where we start using it. That seems like it'll make that commit have more complete context — the string and the use of it complement each other for understanding what's happening.

I think it'd be fine for the first commit to just introduce empty .arb files, and then the next commit can fill them in.